tests/e2e/base/docker-compose.base.yml

```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  # The following services are inherited by three protocols for testing
+  oap: # TODO: bump up OAP version from time to time | 2022-03-25
+    image: ghcr.io/apache/skywalking/oap:547f1ede2d1665601cb9d9a3853ed205620d6c83
+    # Python agent supports gRPC/ HTTP/ Kafka reporting
+    expose:
+      - 11800 # gRPC
+      - 12800 # HTTP
+    networks:
+      - e2e
+    restart: on-failure
```

Comment:
Let's remove this. I don't think we have cases that OAP will fail or cases where we need to restart when it fails, it must be problematic when it fails to start so let's don't waste time waiting for retries.